pkgs/top-level/python-packages.nix

```
@@ -6011,7 +6011,10 @@ in {
 
   packet-python = callPackage ../development/python-modules/packet-python { };
 
-  pwntools = callPackage ../development/python-modules/pwntools { };
+  pwntools = if isPy3k then
+       callPackage ../development/python-modules/pwntools/python3.nix { }
```

probably want to call this 3.nix, follows the convention for most other packages
```suggestion
       callPackage ../development/python-modules/pwntools/3.nix { }
```

---

Hmmm, this convention seems to be for the package version, not the supported Python version...

Example:
  * `pkgs/development/python-modules/pygobject/3.nix` for `pyobject@3.32.1`
  * `pkgs/development/python-modules/pygobject/default.nix` for `pyobject@2.28.6`

---

use `default.nix` for Python 3 version, and `2.nix` for Python 2 version.

---

originally we indeed used the major version number of the package as filename. I started using later on the Python major version as name instead, since we should have only one version of a package anyway.

---

@FRidh there is already a `default.nix`, because a package `pwntools` is already available for python2!

---

currently you have default.nix for the python2 version, but it should be for the one which is expected to be bumped which is the python3 version.

Essentialy what is being asked is:
```
python3.nix -> default.nix
default.nix -> 2.nix
```